utils/nctl/sh/scenarios/common/itst.sh

```diff
@@ -156,6 +156,43 @@ function get_switch_block_equivocators() {
     fi
 }
 
+# Function is used to walk back the blocks to check if a transfer
+# is included under transfer_hashes. If the transfer is not found
+# within the walkback, it will error out.
+function verify_transfer_inclusion() {
```

Comment:
I think we should do similar test for deploys (not transfers).

Reply:
yep, thats up next!
https://casperlabs.atlassian.net/browse/OP-1839